packages/adminrouter/extra/dcos-adminrouter-slave.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Admin Router Agent: A high performance web server and a reverse proxy server
+After=dcos-gen-resolvconf.service
+
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+EnvironmentFile=/opt/mesosphere/environment
+Type=forking
+PIDFile=$PKG_PATH/nginx/logs/nginx.pid
+PrivateDevices=yes
+StandardOutput=journal
+StandardError=journal
+ExecStartPre=/bin/ping -c1 ready.spartan
+ExecStartPre=/bin/ping -c1 leader.mesos
+ExecStartPre=/usr/bin/test -f /var/lib/dcos/cluster-id
+ExecStart=$PKG_PATH/nginx/sbin/nginx -c $PKG_PATH/nginx/conf/nginx.agent.conf
+ExecReload=/usr/bin/kill -HUP \$MAINPID
+KillSignal=SIGQUIT
+KillMode=mixed
+Environment=SECRET_KEY_FILE_PATH=/var/lib/dcos/auth-token-secret
+Environment=AUTH_ERROR_PAGE_DIR_PATH=${PKG_PATH}/nginx/conf/errorpages
+EnvironmentFile=/opt/mesosphere/etc/adminrouter.env
```

Comment:
> I think we might want to wait for Bouncer
There's no bouncer on the agent to wait for. If we wait until bouncer on any/all masters is up, we've created a new dependency between agents and masters.